### ISSUE TITLE
Training model

### ISSUE THREAD

**Issue by WJinhai** [SURFZJY/Real-time-Text-Detection#1]
The trained model can be public? Is it consistent with the results in the paper?

**Comment by SURFZJY** [comment]
The sufflenetv2 model trianed on ic13 and15 has been released, but for some implementation details are different, the results are not as good as the results in the paper. I will continue to update this project with the reference to the [official implementation](https://github.com/MhLiao/DB).

**Comment by WJinhai** [comment]
Does this only work on GPU?

**Comment by SURFZJY** [comment]
It can work on CPU, by setting the gpu_id=None and uncommenting torch.cuda.synchronize(self.device).

**Comment by WJinhai** [comment]
This result looks not as good as in the paper

**Comment by SURFZJY** [comment]
yes, i am working hard to improve the performance. I will release a better version some days later.

**Comment by ghost** [comment]
@SURFZJY Did the results got better?
whats the result on icdar15
